lib.go

```diff
@@ -281,6 +285,16 @@ func SameNicWrapper(config export.L2Info, if1, if2 int) bool {
 	return SameNic(config.GetPtpIfList()[if1], config.GetPtpIfList()[if2])
 }
 
+// Determines if the NIC is intel WPC NIC
+func isWpcNic(ifaceName1 *l2lib.PtpIf) bool {
+	return strings.Contains(ifaceName1.IfPci.Subsytem, WPCNICSubsystemID)
```

Comment:
Subsytem -> Subsystem
add v0.0.5 in go.mod and go mod tidy / go mod vendor
check with make lint